.github/workflows/tests.yml

```diff
@@ -0,0 +1,454 @@
+name: Open Runtimes Tests
+on: 
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  release: 
+    types: [created]
```

Comment:
these can be combined and made more concise

Reply:
Done